src/application.ts

```diff
@@ -200,5 +205,10 @@ export default class Application {
         });
       })
     );
+    const validIncluded = flatten(rawInclude.filter(Boolean));
```

Comment:
@spersico This seems a repetition from `Application#buildOperationResponse`'s first three lines. Should we extract into a single, shared function?

Reply:
noup, I'm a dumbass, I just forgot to delete the other one